.env.example

```diff
@@ -44,3 +47,9 @@ HASURA_GRAPHQL_ADMIN_SECRET=admin-secret
 #REACT_APP_API_BASE_URL=https://coordinape.me/api
 #REACT_APP_HASURA_URL=https://coordinape-prod.hasura.app/v1/graphql
 #REACT_APP_S3_BASE_URL=https://coordinape-prod.s3.amazonaws.com/
+
+
+# These will need to be set to something meaningful for deployment, but in local can be unset
+AWS_ACCESS_KEY_ID=tbd
```

Comment:
We could prefix them with IMAGES_ or something since that is the only use. I can make that change. On your comment about the event handler - It's a good concept but i think its unnecessarily complex. This is a low impact operation (should be couple milliseconds) and it happens very infrequently.

Reply:
Ok I renamed them to be `IMAGES_AWS_X`

Reply:
@CryptoGraffe sure, have no issues with deleting the image in the same mutation, as long as we ensure it doesn't throw an exception if the delete fails

Reply:
Good call, I wrapped it in a try catch